scripts/src/java/org/oppia/android/scripts/common/RepositoryFile.kt

```diff
@@ -0,0 +1,66 @@
+package org.oppia.android.scripts.common
+
+import java.io.File
+
+/** Helper class for managing & accessing files within the project repository. */
+class RepositoryFile() {
+  companion object {
+    /** a list of directories and files which should be excluded for every script check. */
+    val alwaysExcludeFilesList = listOf<String>(
+      ".git",
+      ".gitsecret",
+      ".idea",
+      "bazel",
+      "config",
+      "gradle",
+      ".bazelrc",
+      ".editorconfig",
+      "gradlew",
+      "gradlew.bat",
+      ".aswb",
+    )
+
+    /**
+     * Collects the paths of all the files which are needed to be checked.
+     *
+     * @param repoPath the path of the repo
+     * @param allowedExtension files with only this extension will be included in the search list
+     * @param exemptionsList a list of files which should not be included in the search list
```

Comment:
These are files. Let me make it a bit more clear. `alwaysExcludeDirectoryList` is a list of directories that will always be exempted for each and every check, whereas, `exemptionsList` is an optional argument which provides every script check the liberty to exempt whatever files they want during the analysis. For example: for the case of `TestFileCheck`, when calling this helper method to collect the files, we will pass the `TEST_FILES_EXEMPTIONS_LIST`, so that we don't analyze them for the check. This way, we have made it customizable to work for any check. I hope that this sounds neat. Let me know your thoughts.

Reply:
I think the current version is sufficiently clear--thanks.